gef-ui/server/controllers/amendments/do-you-have-a-facility-end-date/do-you-have-a-facility-end-date.ts

```diff
@@ -0,0 +1,39 @@
+import { CustomExpressRequest } from '@ukef/dtfs2-common';
+import { Response } from 'express';
+import * as api from '../../../services/api';
+import { DoYouHaveAFacilityEndDateViewModel } from '../../../types/view-models/amendments/do-you-have-a-facility-end-date-view-model';
+import { asLoggedInUserSession } from '../../../utils/express-session';
+import { userCanAmendFacility } from '../../../utils/facility-amendments.helper';
+
+export type GetDoYouHaveAFacilityEndDateRequest = CustomExpressRequest<{
+  params: { dealId: string; facilityId: string; amendmentId: string };
+}>;
+
+export const getDoYouHaveAFacilityEndDate = async (req: GetDoYouHaveAFacilityEndDateRequest, res: Response) => {
```

Comment:
documentation please

Reply:
Whoever picks this up, we should update other amendment controllers to do the same, to be consistent.